ftsbot/cogs/antispam.py

```diff
@@ -40,6 +40,45 @@ async def on_message(
 		self,
 		message
 	):
+		if '@everyone' in message.content:
+			has_exception_role = False
+			for role in message.author.roles:
+				if role.name in {
```

Comment:
Shouldn't this use [] instead of {}?

Reply:
you want the same adjusted for lines 111-119 too?

Reply:
yeah, I think only objects should have {} and the existing one was an oversight on my end